tox.ini

```diff
@@ -7,6 +7,8 @@ envlist =
     build_docs
     linkcheck
     codestyle
+requires =
+    tox-uv
```

Comment:
Does this mean we are now using uv for all our package management in tox?

Reply:
yes. This turned out to be the robust solution to ensure that `oldestdeps` wouldn't be run without uv (which would be incorrect). Unfortunately tox doesn't seem to support per-env requirements.

Reply:
Not thrilled by this, what CI jobs do we have left which use pip?

Reply:
I posted here ftr: https://github.com/tox-dev/tox-uv/discussions/135

Reply:
We have the other arch runs that do not use `tox` at all: https://github.com/astropy/astropy/blob/24d3dd96015839ecf87899b02d403a9002bd3ef6/.github/workflows/ci_cron_weekly.yml#L160-L170

(Yes, I put it there, just wanting to be sure we'd get a warning if "standard" installation on a Debian-based system would not work any more. Guess what system I use...)

Reply:
Do we still have to resolve this concern?